### ISSUE TITLE
Incorrect stats on CI build

### ISSUE THREAD

**Issue by ashfurrow** [danger/danger#71]
On [this build](https://circleci.com/gh/artsy/eigen/1619), I got the following output:

```
Danger v0.3.0
DSL Attributes:
------------------
| lines_of_code  | 4
| files_modified | ["Artsy/View_Controllers/Auction/AuctionInformationViewController.swift"]
| files_removed  | []
| files_added    | []
| deletions      | 4
| insertions     | 0
| pr_title       | Adds Swift abstraction for Navigation Buttons
| pr_body        | This abstracts the messiness of working with the `ARNavigationButton` stuff with Swift into one place.
```

Despite the [large-ish patch](https://github.com/danger/danger/files/137143/1619.patch.txt)

**Comment by orta** [comment]
Likely related to https://github.com/danger/danger/issues/68

**Comment by orta** [comment]
okkkkkkk, looks like Danger only cares about one hunk in this diff. So in that case it's this one

<img width="907" alt="screen shot 2016-02-20 at 21 59 06" src="https://cloud.githubusercontent.com/assets/49038/13200586/31aa160e-d81d-11e5-9a3e-9b2396e803a2.png">

**Comment by KrauseFx** [comment]
:dancers:

**Comment by orta** [comment]
should be fixed in #77

**Comment by KrauseFx** [comment]
:dancer:

**Comment by orta** [comment]
Fixed, #88 is the new successor to this bug

**Comment by KrauseFx** [comment]
Oh yeah, @orta goes fast on the fast train :bullettrain_side:
